src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+/**
+ * The Todo class represents a task with no specific deadline or time.
+ * It is a subclass of the Task class.
+ */
+public class Todo extends Task {
+
+    /**
+     * Constructs a Todo object with a description.
+     *
+     * @param description A description of the todo task.
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Converts the Todo object to a string that can be saved to a file.
+     *
+     * @return A formatted string representing the Todo object.
+     */
+    @Override
+    public String toFileString() {
```

Comment:
"toFileString" may be a little difficult to interpret the method's behaviour at a glance. "toSaveableString/Format" could be a better alternative.

Reply:
Personally, ```toFileString``` is still acceptable. However, ```toSaveableFormat``` has better to understand what it is for. Therefore, ```toFileString``` is changed to ```toSaveableFormat```.